docs/section1/Rule 1-6.md

```diff
@@ -0,0 +1,36 @@
+
+# Section 1 - Rule 1-6 
+
+**Rule ID:** 1-6   
+**Rule Description:** On-site renewable energy shall not be included in the baseline building performance.  
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** G3.11 18 Baseline 
+
+**Mandatory Rule:** True    
+**Evaluation Context:** Each RMD  
+**Function Call:**  
+
+## Applicability Check:  
+- All projects are applicable
+
+
+## Rule Logic:
+- get the baseline output schema: `output = B_RMD.output`

```

Comment:
I think we should check this across each orientation, unless that is captured somewhere else. I know Jackson populated the schema for things that should match across the baseline orientations. Not sure if we also did this with with output schema.

Reply:
I addressed this by doing the following:

`- look at each baseline model rotation: `for rotation in [B_RMD, B_RMD_90, B_RMD_180, B_RMD_270]:``